workshop/buildspec-node.yaml

```diff
@@ -0,0 +1,46 @@
+version: 0.2
+
+env:
+    variables:
+      IMAGE_TAG : "v1"
+phases:
+  pre_build:
+    commands:
+      - echo "STARTING PYTHON INSTALLATION"
+      - "curl -s -qL -o python.tgz https://www.python.org/ftp/python/${PY_VERSION}/Python-${PY_VERSION}.tgz"
+      - "tar xf python.tgz -C /usr/bin/"
+      - "python --version"
+      - python -m pip install -U pip
+      - pip install git-remote-codecommit
+      - echo Logging in to CodeCommit
+      - git clone codecommit://appmod-workshop
+      - echo Logging in to Amazon ECR...
+      - aws ecr get-login-password --region $AWS_DEFAULT_REGION | docker login --username AWS --password-stdin $AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com
+  build:
+    commands:
+      - echo Build started on `date`
+      - cd appmod-workshop
+      - echo $( git diff HEAD^ -- applications/node/ | wc -l )
```

Comment:
@YogeshBhatiaAWS 

i did the original buildspec for node with a "diff" with the assumption that we will have a single code pipeline project and that the image only gets build with there are actual commits to the node app.

I see that the java buildspec is performing build regardless, you want to have this be consistent with the java buildspec or is this merge interim ?

Reply:
I agree, I just used the same for the node app, we can change the node app to build irrespective of changes